spec/spec_helper.rb

```diff
@@ -21,7 +22,9 @@ def fixture(name, binary: false, pem: false)
 end
 
 def ssh_keygen_fingerprint(name, algo, priv: false)
-  out = `ssh-keygen #{"-e" if priv} -E #{algo} -l -f #{File.join(FIXTURE_PATH, name)}`
+  out, * = Open3.capture3("ssh-keygen #{'-e' if priv} -E #{algo} -l -f #{File.join(FIXTURE_PATH, name)}")
```

Comment:
Would it make sense to use `capture2` here if we don't care about the output from `err`?

Reply:
Yeah, the reason why I used `capture3` here is because otherwise it doesn't capture stderr, and then the unit tests are filled with that output. We don't need the stderr result, but we also don't need it written out and jumbling the test output.

Reply:
Yeah makes sense